load_sql.go

```diff
@@ -31,23 +31,25 @@ func (t *Tools) LoadSQLQueries(fileName string) (map[string]string, error) {
 func parseSQLQueries(file *os.File, query map[string]string) (map[string]string, error) {
 	scanner := bufio.NewScanner(file)
 	var key string
-	var queries []string
+	var queryBuilder strings.Builder
 	for scanner.Scan() {
 		line := strings.TrimSpace(scanner.Text())
 		if isSQLQuery(line) || len(key) > 0 {
 			if len(key) > 0 {
-				queries = append(queries, line)
 				if strings.HasSuffix(line, ";") {
-					query[key] = strings.Join(queries, " ")
-					key, queries = "", nil
+					queryBuilder.WriteString(line)
+					query[key] = queryBuilder.String()
+					key, queryBuilder = "", strings.Builder{}
+				} else {
+					queryBuilder.WriteString(line + " ")
```

Comment:
was only adding single row sql queries, it has been revised to add multiple row queries.